gdal/port/cpl_conv.cpp

```diff
@@ -2225,6 +2228,11 @@ double CPLDMSToDec( const char *is )
             return tv;
         switch( *s )
         {
+          // UTF-8 "DEGREE SIGN" is 0xC2 0xB0 
+          case static_cast<char>(0xC2):
```

Comment:
would be more elegant to do a switch ( static_cast<unsigned char>(*s) )  instead